atlp-rwanda/vikings-ec-bn
# #184347039 clear cart

### What does this PR do?
Users should be able to clear their cart

### Description of Task to be completed?
Clears products from the cart and reset total to 0

### How should this be manually tested?
1. Checkout to `feature/clear-cart-184347039`
2. Run `npm run seeding`
3. Run `npm run dev`
4. Add products to cart at `http://localhost:<port>/api/v1/cart/add-to-cart` URL in Postman
5. Clear cart at `http://localhost:<port>/api/v1/cart/clear-cart` URL in Postman
6. View cart to check if cleared at `http://localhost:<port>/api/v1/cart/view-cart` URL in Postman
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#184347039](https://www.pivotaltracker.com/story/show/184347039)
